CLIENT-SPECIFICATION.md

```diff
@@ -213,7 +213,7 @@ Step  | Path checked         | Outcome
 
 ## Caching
 
-If appropriate, it is RECOMMENDED that clients implement a cache of pages. If implemented, clients MUST download the entire archive either as a whole from **[https://tldr.sh/assets/tldr.zip](https://tldr.sh/assets/tldr.zip)** (Which redirects to [https://raw.githubusercontent.com/tldr-pages/tldr-pages.github.io/main/assets/tldr.zip](https://raw.githubusercontent.com/tldr-pages/tldr-pages.github.io/main/assets/tldr.zip)) or download language-specific translation archives in the format `https://tldr.sh/assets/tldr-pages.{{language-code}}.zip` (Which redirects to [https://raw.githubusercontent.com/tldr-pages/tldr-pages.github.io/main/assets/tldr-pages.{{language-code}}.zip](https://raw.githubusercontent.com/tldr-pages/tldr-pages.github.io/main/assets)), along with the archive for English from **[https://tldr.sh/assets/tldr-pages.zip](https://tldr.sh/assets/tldr-pages.zip)** (It redirects to [https://raw.githubusercontent.com/tldr-pages/tldr-pages.github.io/main/assets/tldr-pages.zip](https://raw.githubusercontent.com/tldr-pages/tldr-pages.github.io/main/assets/tldr-pages.zip)).
+If appropriate, it is RECOMMENDED that clients implement a cache of pages. If implemented, clients MUST download the entire archive either as a whole from **<https://tldr.sh/assets/tldr.zip>** or download language-specific archives in the format `https://tldr.sh/assets/tldr-pages.{{language-code}}.zip` (e.g. the English archive can also be downloaded from **<https://tldr.sh/assets/tldr-pages.zip>**).
```

Comment:
I personally would suggest discouraging the other one, to have consistent language code format files (but some clients don't support additional languages at all so IG its fine).

Reply:
Not sure what you mean @kbdharun? Which other one? `https://tldr.sh/assets/tldr-pages.zip` implies to me to be a .zip containing all possible pages, and `https://tldr.sh/assets/tldr-pages.en.zip` clearly indicates English-only pages?

Reply:
> Not sure what you mean @kbdharun? Which other one?

I meant this aliased one, <https://tldr.sh/assets/tldr-pages.zip> it doesn't follow the language structure so do we need to recommend it now that we are updating the whole section?

Reply:
Ahh okay. Yeah, that makes sense. But the current spec says <https://tldr.sh/assets/tldr.zip> is for all languages rather than just english?

But yes, we should probably rework this to recommend GitHub releases, as in #12158.